Makefile

```diff
@@ -73,3 +73,24 @@ push-tags:
 	@git tag -a ${TAG} -s -m "Version ${TAG}"
 	@echo "Pushing tag ${TAG}"
 	@git push ${REMOTE} ${TAG}
+
+# Used for debug only
```

Comment:
Is this comment correct? If those lines are really just for debug, should we remove them?

Reply:
While it's not used for production release, it was making it much easier to troubleshoot and add/modify the build/release workflows (more easily pushing and deleting/repeating tag push to trigger the appropriate workflows in my fork), so I thought it appropriate/helpful to leave them for other devs.